ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh

```diff
@@ -12,10 +12,10 @@ CSVNAME="$(oc get packagemanifests/gpu-operator-certified -n openshift-marketpla
 sed -i -e "0,/v1.11/s//$CHANNEL/g" -e "s/gpu-operator-certified.v1.11.0/$CSVNAME/g"  "$GPU_INSTALL_DIR/gpu_install.yaml"
 
 oc apply -f "$GPU_INSTALL_DIR/gpu_install.yaml"
-
+oc apply -f "$GPU_INSTALL_DIR/nfd_operator.yaml"
```

Comment:
for NVIDIA GPUs, NFD operator gets deployed by NVIDIA operator right?

Reply:
The NFD operator  is an independent operator and is not installed  by the Nvidia operator.